go/action_kit_sdk/prepare_helper/prepare_helper.go

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: MIT
+// SPDX-FileCopyrightText: 2023 Steadybit GmbH
+
+package prepare_helper
```

Comment:
I'd love to see this not in an extra package. 
I guess we'd then need the PrepareResult/StopResult/... have a common struct, so that we'd not have clashing names, right?

Reply:
Since the SDK is already in a separate package, we could just move them `action_kit_sdk`